1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,45 @@
+import { fizzbuzz } from './fizzbuzz';
 
-describe("fizzbuzz", () => {
+describe('fizzbuzz', () => {
+  describe('Accept only input of type number, between 1 and 100, and returns string', () => {
+    it('throws an error when given non-number input', () => {
+      expect(() => fizzbuzz('hello' as any)).toThrow();
```

Comment:
**question(non-blocking)**: We might be testing typescript here, even though its in the checklist I'm not sure it makes sense to test and hence implement this one...